pipeline/transformer/transformer_marc_xml/src/main/scala/weco/pipeline/transformer/marc/xml/transformers/MarcXMLRecordTransformer.scala

```diff
@@ -0,0 +1,43 @@
+package weco.pipeline.transformer.marc.xml.transformers
+
+import weco.catalogue.internal_model.identifiers.{
+  DataState,
+  IdentifierType,
+  SourceIdentifier
+}
+import weco.catalogue.internal_model.work.WorkState.Source
+import weco.catalogue.internal_model.work.{Work, WorkData}
+import weco.pipeline.transformer.marc.xml.data.MarcXMLRecord
+import weco.pipeline.transformer.marc_common.transformers.MarcTitle
+
+import java.time.Instant
+
+object MarcXMLRecordTransformer {
+
+  def apply(record: MarcXMLRecord): Work.Visible[Source] = {
+    // TODO: The state stuff here is EBSCO-specific.  Move it when the EBSCO transformer is implemented
```

Comment:
Perhaps we should just deal with this as an "EBSCO Transformer" until we have cause to deal with another MARCXML source?

Reply:
I thought of this, and it might turn out that way.  I started writing this as EBSCO-specific, then made it purely source-independent thing, but then realised I needed an id with an IdentifierType, so this bit of EBSCO had to stay.

Ideally, as with the Sierra changes, I'd like us to understand the difference between the quirks of a source and the general behaviour of a format.

Reply:
> so this bit of EBSCO had to stay.

Interestingly I wondered whether pulling the source identifier from control field 001 is something that might be source independent if we had multiple sources. Given the field definition as:

> For interchange purposes, documentation of the structure of the control number and input conventions should be provided to exchange partners by the organization initiating the interchange.

We don't do this for Sierra but it looks like the bnumber is in varField 001 for those records created in Sierra.

> understand the difference between the quirks of a source and the general behaviour of a format

Would we treat MARCXML any differently from MARC21 in terms of general behaviour? My mental model is we are parsing MARC-ish data from multiple sources (Sierra/EBSCO) into our internal representation of MARC and we can pick which fields from a source need special treatment otherwise falling back to the "common" MARC transform for that field. 

Hope that makes sense.

Reply:
I think pulling stuff out of 001 should be source-independent, but calling it an EbscoAltLookup is (obvs) EBSCO-specific.   I also think that trying to find the last modified time somewhere and falling back to some default that isn't in the record itself (Now, or file creation time) should also be source-independent.